bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/LedgerMetadataSerDe.java

```diff
@@ -78,7 +78,7 @@
     public static final int METADATA_FORMAT_VERSION_3 = 3;
 
     public static final int MAXIMUM_METADATA_FORMAT_VERSION = METADATA_FORMAT_VERSION_3;
-    public static final int CURRENT_METADATA_FORMAT_VERSION = METADATA_FORMAT_VERSION_2;
+    public static final int CURRENT_METADATA_FORMAT_VERSION = METADATA_FORMAT_VERSION_3;
```

Comment:
for now I changed the metadata version. I'm open for feedback, regarding if it is ok to bump hardcoded metadata version, make it configurable or something else..

Reply:
@sijie just want to confirm that you are fine with this hardcoded value change.